Metadata review - AUVs

cdobs opened the pull request:
Note: Early cal files for PARS (ie: SN50170_20140109) -> There are no values listed for wet scale factors. The values for dry were present in the original cal csvs.

Still todo:
1. Use a script to compare NUTNR cals with cal csvs
2. Track down where ADCPA-N documentation lives.
3. Confirm that using dry scale factors in place of wet in cal csv for pars 50170/50180 early cals is okay.

rtravis-whoi commented on calibration/DOSTAN/CGINS-DOSTAN-01519__20180619.csv:
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+1519,CC_calibration_date,1529366400000,sensor calibration date (milliseconds since 1970-01-01)
```
Looks good.

pbrickley replied:
Good

rtravis-whoi commented on calibration/DOSTAN/CGINS-DOSTAN-01520__20180119.csv:
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+1520,CC_calibration_date,1516320000000,sensor calibration date (milliseconds since 1970-01-01)
```
Cal date of filename should be 20180129 (not 20180119).  milliseconds since 1970-01-01: 1517184000000

pbrickley replied:
Agreed.  The cal cert file is dated 2018-01-29

rtravis-whoi commented on calibration/FLORTN/CGINS-FLORTN-01108__20180221.csv:
```diff
@@ -0,0 +1,5 @@
+1108,CC_angular_resolution,1.076,
+1108,CC_depolarization_ratio,0.039,
+1108,CC_measurement_wavelength,700,
+1108,CC_scattering_angle,124,
```
Looks good.

pbrickley replied:
OK

rtravis-whoi commented on calibration/FLORTN/CGINS-FLORTN-01109__20180619.csv:
```diff
@@ -0,0 +1,5 @@
+1109,CC_angular_resolution,1.076,
+1109,CC_depolarization_ratio,0.039,
+1109,CC_measurement_wavelength,700,
+1109,CC_scattering_angle,124,
```
Looks good.

pbrickley replied:
ok

rtravis-whoi commented on calibration/PARADN/CGINS-PARADN-50170__20140109.csv:
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-50170,CC_dark_offset,9.4,in mV
+50170,CC_dark_offset,9.4,in mV; source_file PARAD-N_QSP2150A_SN_50170_Calibration_Certificate_2014-01-09.pdf
```
Source file correct.

snwhite replied:
Looks good.

rtravis-whoi commented on calibration/PARADN/CGINS-PARADN-50170__20180206.csv:
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+50170,CC_dark_offset,9.4,in mV; source_file PARAD-N_QSP2150A_SN_50170_Calibration_Certificate_2018-02-06.pdf
+50170,CC_scale_wet,7.242,in Volts/(uE/cm^2sec)
```
Looks good.

snwhite replied:
Looks good.

rtravis-whoi commented on calibration/PARADN/CGINS-PARADN-50180__20140109.csv:
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-50180,CC_dark_offset,10.6,in mV
+50180,CC_dark_offset,10.6,in mV; source_file PARAD-N_QSP2150A_SN_50180_Calibration_Certificate_2014-01-09.pdf
```
source file correct.

snwhite replied:
Looks good.

rtravis-whoi commented on calibration/PARADN/CGINS-PARADN-50180__20180716.csv:
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+50180,CC_dark_offset,10.6,in mV; source_file PARAD-N_QSP2150A_SN_50180_Calibration_Certificate_2018-07-16.pdf
+50180,CC_scale_wet,5.963,in Volts/(uE/cm^2sec)
```
Looks good.

snwhite replied:
Looks good.

rtravis-whoi commented on calibration/PARADN/CGINS-PARADN-50267__20171116.csv:
```diff
@@ -1,3 +1,3 @@
-50267,CC_dark_offset,10.4,in mV
-50267,CC_scale_wet,6.434,in Volts/(uE/cm^2sec)
+50267,CC_dark_offset,10.1,in mV; source_file PARAD-N_QSP2150A_SN_50267_Calibration_Certificate_2017-11-16.pdf
+50267,CC_scale_wet,5.604,in Volts/(uE/cm^2sec)
```
Looks good.

snwhite replied:
Looks good.

rtravis-whoi commented on calibration/PARADN/CGINS-PARADN-50267__20190617.csv:
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+50267,CC_dark_offset,10.1,in mV; source_file PARAD-N_QSP2150A_SN_50267_Calibration_Certificate_2019-06-17.pdf
+50267,CC_scale_wet,4.815,in Volts/(uE/cm^2sec)
```
Looks good.

snwhite replied:
Looks good.

rtravis-whoi commented on calibration/PARADN/CGINS-PARADN-50266__20171116.csv:
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-50266,CC_dark_offset,10.4,in mV
+50266,CC_dark_offset,10.4,in mV; source_file PARAD-N_QSP2150A_SN_50266_Calibration_Certificate_2017-11-16.pdf
 50266,CC_scale_wet,6.434,in Volts/(uE/cm^2sec)
```
Looks good.

snwhite replied:
Looks good.

cdobs commented on calibration/CTDAVN/CGINS-CTDAVN-00424__20171006.csv:
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+424,CC_calibration_date,1507248000000,sensor calibration date (milliseconds since 1970-01-01)
```
Renamed to resolve @rtravis-whoi comment about FN. I think her comments went away since I renamed it via the browser

rtravis-whoi replied:
looks good

cdobs commented on calibration/CTDAVN/CGINS-CTDAVN-00425__20171006.csv:
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+425,CC_calibration_date,1507248000000,sensor calibration date (milliseconds since 1970-01-01)
```
Renamed to resolve @rtravis-whoi comment about FN. I think her comments went away since I renamed it via the browser

snwhite replied:
Looks good.

rtravis-whoi replied:
looks good

cdobs commented on calibration/DOSTAN/CGINS-DOSTAN-01520__20180129.csv:
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+1520,CC_calibration_date,1517184000000,sensor calibration date (milliseconds since 1970-01-01)
```
Resolving @rtravis-whoi comment to correct cal date in FN and in csv to 2018-01-29.

rtravis-whoi replied:
looks good.

cdobs commented:
@rtravis-whoi I addressed the fixes you mentioned and am requesting you review the changes I made for them. I also updated the PR with my NUTNR-N and ADCPA-N review fixes. Those should be reviewed 2x as well.

cdobs commented:
Still need to address #3 on the todo list at the top of the PR: 
_3. Confirm that using dry scale factors in place of wet in cal csv for pars 50170/50180 early cals is okay.._ 

#1 and #2 were resolved.

pbrickley commented on calibration/CTDAVN/CGINS-CTDAVN-00424__20171009.csv:
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-424,CC_calibration_date,1507507200000,sensor calibration date (milliseconds since 1970-01-01)
+424,CC_calibration_date,1507248000000,sensor calibration date (milliseconds since 1970-01-01)
```
The date of the cal file is 10/06/2017, but the date of the calibration is 9/28/2017.  Is this not then 1506556800000?

pbrickley commented on calibration/CTDAVN/CGINS-CTDAVN-00424__20170928.csv:
```diff
@@ -1,2 +0,0 @@
-serial,name,value,notes
-424,CC_calibration_date,1506556800000,sensor calibration date (milliseconds since 1970-01-01)
```
The file certification date is 10/06/2017.  The 9/28/2017 comes from the CT cal page.  Probably we should use the cal cert file date 10/06/2017

pbrickley commented on calibration/CTDAVN/CGINS-CTDAVN-00425__20171009.csv:
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-425,CC_calibration_date,1507507200000,sensor calibration date (milliseconds since 1970-01-01)
+425,CC_calibration_date,1507248000000,sensor calibration date (milliseconds since 1970-01-01)
```
Agreed.

snwhite commented on calibration/CTDAVN/CGINS-CTDAVN-00424__20171006.csv:
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+424,CC_calibration_date,1507248000000,sensor calibration date (milliseconds since 1970-01-01)
```
Looks good.

snwhite commented on calibration/NUTNRN/CGINS-NUTNRN-01121__20171211.csv:
```diff
@@ -1,5 +1,5 @@
+1121,CC_cal_temp,19.95, source_file NUTNR-N_SUNA_SN_1121_Calibration_Files_2017-12-11.zip > SNA1121B.cal
 1121,CC_di,"[34, 28, 48, 32, 64, 70, 66, 63, 70, 62, 78, 63, 70, 93, 423, 1964, 5938, 12559, 20441, 27880, 33701, 37455, 39226, 39416, 38498, 37045, 35470, 34018, 32926, 32222, 31921, 32122, 32741, 33789, 35329, 37289, 39623, 42362, 45333, 48320, 51181, 53596, 55330, 56231, 56074, 54974, 53232, 50938, 48493, 46108, 43894, 42066, 40614, 39535, 38860, 38540, 38517, 38805, 39377, 40184, 41258, 42574 ...
 1121,CC_eno3,"[0.00098736, 0.01255601, 0.00486392, 0.00313038, 0.00702902, 0.00158067, 0.00265375, -0.00049719, -0.00048258, 0.00044672, 0.00159068, -0.00236896, -0.00462136, 0.00406782, 0.00123471, 0.00220968, 0.00265691, 0.00486158, 0.00273392, 0.00237544, 0.00170665, 0.00235385, 0.00092791, 0.00385255, 0.00583814, 0.00680752, 0.0064489, 0.00675755, 0.00660485, 0.00638027, 0.00609724, 0.0058523 ...
 1121,CC_eswa,"[0.00173941, 0.0002355, -0.00069074, 0.00012665, 0.00492102, 0.00668267, 0.00053095, 0.00505082, 0.005768, 0.00205489, 0.00807245, 0.00788015, 0.01089029, 0.00822573, 0.02903903, 0.04554747, 0.05804202, 0.06521274, 0.07294347, 0.07844077, 0.07861915, 0.07995316, 0.07883529, 0.0738619, 0.06623207, 0.05832528, 0.04980367, 0.04185513, 0.03467389, 0.02846651, 0.02322254, 0.01875361, 0.0 ...
```
Source file correct.

rtravis-whoi replied:
looks good.

snwhite commented on calibration/NUTNRN/CGINS-NUTNRN-01109__20171211.csv:
```diff
@@ -1,5 +1,5 @@
+1109,CC_cal_temp,20.00, source_file NUTNR-N_SUNA_SN_1109_Calibration_Files_2017-12-11.zip > SNA1109C.cal
 1109,CC_di,"[47, 50, 50, 34, 42, 30, 55, 40, 61, 109, 308, 888, 2288, 4904, 8621, 12795, 16661, 20035, 23027, 25922, 28792, 31703, 34643, 37483, 40040, 42096, 42776, 44076, 43816, 42910, 41393, 39631, 37819, 36059, 34546, 33292, 32355, 31768, 31520, 31616, 32052, 32816, 33821, 35222, 36912, 38853, 41073, 43432, 45908, 48350, 50597, 52531, 54034, 54898, 55107, 54680, 53572, 51910, 49971, 47731, 45 ...
 1109,CC_eno3,"[0.00659662, 0.00376627, 0.01333674, 0.00539796, -0.00212481, -0.00481451, -0.00646904, -0.00449379, 0.00062205, 0.00299681, 0.00264381, 0.00267459, 0.00288843, 0.01075125, 0.00654761, 0.00084272, -0.00247749, -0.00586457, 0.00554414, 0.0024195, 0.00397617, 0.00698356, 0.00705519, 0.00673591, 0.00674074, 0.00652793, 0.00642625, 0.00615741, 0.00591098, 0.00564849, 0.00538508, 0.00506 ...
 1109,CC_eswa,"[0.00433451, 0.00635477, 0.00680021, 0.00027497, 0.01165361, 0.0028623, 0.01104832, 0.00648852, 0.01025788, 0.01535008, 0.02883005, 0.04057657, 0.05043968, 0.06245075, 0.07161601, 0.07490654, 0.0808423, 0.08526647, 0.07644731, 0.07489033, 0.07079746, 0.06380919, 0.05557308, 0.04783547, 0.04063115, 0.0342027, 0.02861453, 0.02378727, 0.01956914, 0.01598603, 0.01294041, 0.01041988, 0.0 ...
```
Source file correct.

rtravis-whoi replied:
looks good

snwhite commented on calibration/NUTNRN/CGINS-NUTNRN-00367__20131016.csv:
```diff
@@ -1,5 +1,5 @@
+367,CC_cal_temp,20.03, source_file NUTNR-N_SUNA_SN_0367_Calibration_Files_2013-10-16.zip > SNA0367A.cal
 367,CC_di,"[29, 28, 31, 18, 34, 31, 38, 23, 21, 34, 11, 35, 73, 244, 872, 2608, 5883, 10428, 15174, 19075, 21638, 23023, 23513, 23535, 23366, 23303, 23315, 23612, 24163, 25036, 26171, 27426, 29546, 31716, 34184, 36852, 39653, 42412, 44919, 46970, 48363, 48940, 48726, 47728, 46185, 44245, 42179, 40206, 38460, 36946, 35789, 35017, 34458, 34249, 34338, 34661, 35240, 36031, 37038, 38248, 39618, 41122 ...
 367,CC_eno3,"[-0.0015426, -0.00404227, -0.00078705, 0.01063911, 0.00604444, 0.0045979, -0.00528552, -0.00921668, -0.00417808, -0.00168447, 0.00163443, 0.00132554, 0.00479796, 0.00371854, -0.00021844, 0.00605293, 0.00240645, -0.00020868, 0.00489757, -0.00235004, -0.01934684, 0.00781271, 0.00420987, 0.00179037, 0.0047007, 0.00768489, 0.00748915, 0.00720286, 0.00708183, 0.00699254, 0.00653209, 0.006 ...
 367,CC_eswa,"[-0.00157831, 0.00174518, 0.00548307, -0.0069939, 0.00376812, 0.0025558, 0.00691596, 0.00625939, 0.00330973, 0.0114033, -0.0074477, 0.00176782, 0.01282124, 0.0216845, 0.043205, 0.0500214, 0.07006668, 0.07713425, 0.08988811, 0.08378474, 0.10745538, 0.09060502, 0.08296636, 0.08122516, 0.06623853, 0.05478994, 0.04607975, 0.03842684, 0.03167692, 0.02589917, 0.02108815, 0.0170452, 0.01370 ...
```
Source file correct.

rtravis-whoi replied:
looks good.

snwhite commented on calibration/NUTNRN/CGINS-NUTNRN-00366__20131006.csv:
```diff
@@ -1,5 +1,5 @@
+366,CC_cal_temp,20.03, source_file NUTNR-N_SUNA_SN_0366_Calibration_Files_2013-10-16.zip > SNA0366A.cal
 366,CC_di,"[40, 42, 35, 29, 18, 30, 48, 30, 32, 42, 71, 150, 528, 1617, 4036, 8133, 13286, 18322, 22114, 24347, 25278, 25374, 25031, 24621, 24321, 24210, 24438, 24944, 25793, 27003, 28583, 30495, 32722, 35232, 37896, 40578, 43192, 45432, 47167, 48223, 48455, 47945, 46740, 45022, 43119, 41127, 39227, 37657, 36341, 35330, 34743, 34413, 34397, 34612, 35137, 35866, 36811, 37919, 39235, 40698, 42251,  ...
 366,CC_eno3,"[-0.00385557, 0.00198775, -0.00026332, -0.0272395, -0.00025662, -0.00025327, -0.00777571, -0.00434304, -0.00825164, -0.00200444, -0.00277301, -0.00227497, 0.0028936, 0.0053197, 0.00400543, 0.00279457, -0.00089524, -0.00560307, 0.00397755, -0.00593976, -0.00062063, 0.0010983, 0.00098502, 0.00535332, 0.00758353, 0.00692391, 0.00728361, 0.00709547, 0.00675502, 0.00649439, 0.00614732, 0. ...
 366,CC_eswa,"[0.00701215, 0.01242737, 0.00470067, 0.03490335, -0.00025637, -0.00068557, 0.01504146, 0.00268351, 0.01379215, 0.01164581, 0.0170885, 0.02136508, 0.03507115, 0.05218687, 0.06378461, 0.07217749, 0.07945466, 0.08502082, 0.07987227, 0.0913495, 0.08137411, 0.07707559, 0.07279577, 0.06211515, 0.05242061, 0.04429593, 0.03683581, 0.03035994, 0.0249061, 0.02024738, 0.01636294, 0.01314408, 0. ...
```
Source file correct.

rtravis-whoi replied:
CSV filename date should be 20131016
